start.sh

```diff
@@ -11,18 +11,18 @@ fi
 
 
 docker run --rm --name pxeserver --privileged \
```

Comment:
Is `--privileged` still needed?

Reply:
Needs verification. I had lots of problems with network in docker. I have read on some sites that this option helps much.